### PR TITLE
Answers for exercises 9,10,11 for Tuesday Week 2

### PR DESCRIPTION
**Pull request by JohnLoftin**

### CODE REVIEW

File: week2/1-tuesday/class/exercise10/exercise10.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+function fiveOrThree(number){
+    if(number % 5 === 0 || number % 3 === 0){
+        return true;
+    } else {
+        return false;
+    }
+}
```
Comment (samuraijane) [inline week2/1-tuesday/class/exercise10/exercise10.js:7]: This works but there are two edge cases it won't account for:

- something other than a number is passed in
- a zero is passed in

Update and test with this:

```suggestion
    if(typeof number !== "number") return 'Sorry, you must use a number. Please try again.'
    if(!number) return 'Sorry, we do not accept zeros';
    if(number % 5 === 0 || number % 3 === 0){
        return true;
    } else {
        return false;
    }
}
```

The `!number` may be confusing but it's testing to see if `number` is false and `0` is considered `false`.

File: week2/1-tuesday/class/exercise11/exercise11.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const randomString = "Random String";
+
+function checkForSorE(string){
+    var lowerCase = string.toLowerCase();
```
Comment (samuraijane) [inline week2/1-tuesday/class/exercise11/exercise11.js:4]: Very good, minor suggestion for error checking:

```suggestion
function checkForSorE(userInput) {
    if(typeof userInput !== "string") return `Sorry, you must enter a string, not a ${typeof userInput}`;
    var lowerCase = userInput.toLowerCase();
```

I've renamed `string` to `userInput` to make it easier to follow what we're doing when the input is not actually a `string`. It's fine to leave it how you did it but this might save someone a few seconds when they read it for the first time.

File: week2/1-tuesday/class/exercise9/exercise9.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function tenAndTwenty(age){
+    if(age > 10 && age < 20){
```
Comment (samuraijane) [inline week2/1-tuesday/class/exercise9/exercise9.js:2]: Well, done. See if you can add the error check on your own for this one (the code that makes sure it's actually dealing with a number. One thing to keep in mind on this is if someone enters a "5", for example, it's technically a string but you can still use it as long as you can cast it to a number (which you can in this case).

**Review by samuraijane — CHANGES_REQUESTED** [review]
Great job John! I left some comments for each exercise. Please review and resubmit and let me know if you have any questions.
